hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -4,16 +4,17 @@
 
 This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or a USB hard disk. Note that this feature does not work with all USB mass storage devices.
 
-See the [documentation](./) for the boot sequence and alternative boot modes (Network, USB device, GPIO or SD boot).
+See the [documentation](README.md) for the boot sequence and alternative boot modes (Network, USB device, GPIO or SD boot).
```

Comment:
I still think the link-text should be "bootmodes documentation" rather than just "documentation" :wink:

Reply:
Agreed and edited.